packages/stripe_web/lib/src/widgets/payment_element.dart

```diff
@@ -10,7 +10,14 @@ import 'package:web/web.dart' as web;
 import '../../flutter_stripe_web.dart';
 
 export 'package:stripe_js/stripe_api.dart'
-    show PaymentElementLayout, ElementAppearance, ElementTheme;
+    show
+        PaymentElementLayout,
+        ElementAppearance,
+        ElementTheme,
+        ElementAppearanceLabels;
+
+// TODO: should not use src of other package
+export 'package:stripe_js/src/api/elements/payment_element_options.dart';
```

Comment:
There are some overlaps between stripe_js/api and platform_interface so this file is exported instead. It is not recommended to do this this way. Feel free to not merge this and fix the overlaps instead.